ivy/functional/backends/jax/random.py

```diff
@@ -25,8 +25,13 @@ def random_uniform(
     *,
     device: jaxlib.xla_extension.Device,
     dtype: jnp.dtype,
-    out: Optional[JaxArray] = None
 ) -> JaxArray:
+    if isinstance(low, jaxlib.xla_extension.DeviceArray):
```

Comment:
could you please add some comments explaining why this is necessary? The [docs](https://jax.readthedocs.io/en/latest/_autosummary/jax.random.uniform.html#jax.random.uniform) seems to suggest that these are permitted to be arrays.

Reply:
There was some underlying issue which required me to add in these lines just as a temporary fix.
They aren't needed anymore so I'll remove them in the next commit!